test/integration/run.sh

```diff
@@ -25,7 +25,7 @@ export HELM_DATA_HOME="${helm_dir}/data"
 export HELM_HOME="${HELM_DATA_HOME}"
 export HELM_PLUGINS="${HELM_DATA_HOME}/plugins"
 export HELM_CONFIG_HOME="${helm_dir}/config"
-HELM_SECRETS_VERSION=3.5.0
+HELM_SECRETS_VERSION=${HELM_SECRETS_VERSION}
```

Comment:
@KqLLL set a default version will be better when HELM_SECRETS_VERSION is null. WDYT?